sos/cleaner/mappings/hostname_map.py

```diff
@@ -80,10 +80,6 @@ def load_domains_from_map(self):
                         self._domains[_domain_to_inject] = _ob_domain
         self.set_initial_counts()
 
-    def load_domains_from_options(self, domains):
-        for domain in domains:
-            self.sanitize_domain(domain.split('.'))
-
```

Comment:
How is this achieved in the new code, please..?

Reply:
https://github.com/sosreport/sos/pull/3262/files#diff-affa4239dfe2658d3429ab817a6242bcab87ecb4e0959036e87b20ae2133c6e1R60-R61

Reply:
Ah I see, now it is more straighforward.